test/unit/spec/EnrollWebauthn_spec.js

```diff
@@ -136,50 +123,68 @@ function (Okta,
 
     Expect.describe('Enroll factor', function () {
       itp('shows error if browser does not support webauthn', function () {
-        spyOn(webauthn, 'isWebauthnOrU2fAvailable').and.returnValue(false);
+        spyOn(webauthn, 'isNewApiAvailable').and.returnValue(false);
 
         return setup().then(function (test) {
           expect(test.form.errorHtml()).toHaveLength(1);
-          expect(test.form.errorHtml().html()).toEqual('Security Key (U2F) is not supported on this browser.' +
+          expect(test.form.errorHtml().html()).toEqual('Security key or built-in authenticator is not supported on this browser.' +
             ' Select another factor or contact your admin for assistance.');
         });
       });
 
       itp('shows error if browser does not support webauthn and only one factor', function () {
-        spyOn(webauthn, 'isWebauthnOrU2fAvailable').and.returnValue(false);
+        spyOn(webauthn, 'isNewApiAvailable').and.returnValue(false);
 
         return setup(false, true).then(function (test) {
           expect(test.form.errorHtml()).toHaveLength(1);
-          expect(test.form.errorHtml().html()).toEqual('Security Key (U2F) is not supported on this browser.' +
+          expect(test.form.errorHtml().html()).toEqual('Security key or built-in authenticator is not supported on this browser.' +
             ' Contact your admin for assistance.');
         });
       });
 
       itp('does not show error if browser supports webauthn', function () {
-        spyOn(webauthn, 'isWebauthnOrU2fAvailable').and.returnValue(true);
+        spyOn(webauthn, 'isNewApiAvailable').and.returnValue(true);
         return setup().then(function (test) {
           expect(test.form.errorHtml()).toHaveLength(0);
         });
       });
 
       itp('shows instructions and a register button', function () {
-        spyOn(webauthn, 'isWebauthnOrU2fAvailable').and.returnValue(true);
+        spyOn(webauthn, 'isNewApiAvailable').and.returnValue(true);
         return setup().then(function (test) {
           Expect.isVisible(test.form.enrollInstructions());
           Expect.isVisible(test.form.submitButton());
         });
       });
 
-      itp('shows a waiting spinner and devices images after submitting the form', function () {
+      itp('shows correct instructions for Edge browser', function () {
+        const userAgent = window.navigator.userAgent;
+        spyOn(webauthn, 'isNewApiAvailable').and.returnValue(true);
+        window.navigator['__defineGetter__']('userAgent', () => {
```

Comment:
once we create a function in BrowserFeatures, this could be as simple as `spyOn(BrowserFeatures, 'isEdge').and.returnValue(true);`